ui/pages/v2/HelmChartDetail.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import styled from "styled-components";
+import Interval from "../../components/Interval";
+import Page, { Content, TitleBar } from "../../components/Page";
+import SourceDetail from "../../components/SourceDetail";
+import { HelmChart, SourceRefSourceKind } from "../../lib/api/core/types.pb";
+
+type Props = {
+  className?: string;
+  name: string;
+  namespace: string;
+};
+
+function HelmChartDetail({ className, name, namespace }: Props) {
+  return (
+    <Page error={null} className={className}>
+      <SourceDetail
+        name={name}
+        namespace={namespace}
+        type={SourceRefSourceKind.HelmChart}
+        info={(ch: HelmChart) => [
+          ["Chart", ch?.chart],
+          ["Ref", ch?.sourceRef?.name],
+          ["Last Updated", ""],
```

Comment:
We should probably have a `<Timestamp />` value here.

Reply:
Just added Timestamps wherever last updated was blank